Makefile

```diff
@@ -69,14 +69,14 @@ KUSTOMIZE_VERSION := v5.4.2-gke.0
 KUSTOMIZE := $(BIN_DIR)/kustomize
 KUSTOMIZE_STAGING_DIR := $(OUTPUT_DIR)/third_party/kustomize
 
-HELM_VERSION := v3.14.4-gke.2
+HELM_VERSION := v3.15.3-gke.0
```

Comment:
Can you move the helm update to a different PR?
Best to keep these seperate in case we need to revert.

Reply:
Done, modified commit and title.